tests/BinaryOperators/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -168,11 +168,7 @@ contract ArithmeticOperators {
             veryVeryVeryVeryVeryLongVariableCalledB;
         veryVeryVeryVeryVeryLongVariableCalledA +
             veryVeryVeryVeryVeryLongVariableCalledB -
-            c *
-            d /
-            e %
-            f **
-            g;
+            (((c * d) / e) % f ** g);
```

Comment:
I think `(c * d / e) % f ** g` might be a little better, but I don't know how hard that would be to do.

Reply:
it has parentheses because it's following `veryVeryVeryVeryVeryLongVariableCalledB -`
if you try prettier `veryVeryVeryVeryVeryLongVariableCalledA + veryVeryVeryVeryVeryLongVariableCalledB - c * d / e % f ** g` in javascript it will create the same parentheses.
`+` and `-` are well known to have little precedence but modulo `%` is a tricky one that's why I think prettier js wraps the `%` between parentheses, you can check this rule in lines 43 and 44 of `src/parser.js` in this PR.
Anyway, we don't have to follow by the letter each rule of prettier js, I just thought it was a good place to start, we can add or remove rules according to what makes more sense. My main concern is that the parentheses suggestions have to follow the order of precedence of operators, otherwise, the bytecode will not be the same when compiled.

Reply:
Sorry, I mean specifically the parentheses around `c*d`. I think `c * d / e` is better than `(c * d) / e`, but that's just me. Besides, if that's how prettierjs does it, I'm more than fine with following its lead.